webapp/app.py

```diff
@@ -29,5 +29,16 @@ def index():
 
 
 @app.route("/charm")
-def details():
-    return render_template("details.html")
+def charm():
```

Comment:
Can you add a quick note that when we have the api and proper routes, these two endpoints and methods will be the same `details` endpoint again?

We could potentially change it to be "/details" with a "?type=charm|bundle" for now but I don't think that needs to be done in this PR